mmv1/third_party/terraform/services/logging/resource_logging_bucket_config.go

```diff
@@ -405,3 +458,40 @@ func flattenCmekSettings(cmekSettings interface{}) []map[string]interface{} {
 
 	return []map[string]interface{}{data}
 }
+
+func expandIndexConfigs(originalIndexConfigs []interface{}) ([]map[string]interface{}, error) {
+	transformedIndexConfigs := make([]map[string]interface{}, 0, len(originalIndexConfigs))
+	for _, entry := range originalIndexConfigs {
+		original := entry.(map[string]interface{})
+		var validIndexType = false
```

Comment:
Can we just defer to the API error here? We're already in apply, let's not add extra complexity clientside.

If this was a ValidateFunc- which works at plan time- it may be worth keeping around.